fbs/response.fbs

```diff
@@ -0,0 +1,21 @@
+include "router.fbs";
+include "transport.fbs";
+include "webRtcServer.fbs";
+include "worker.fbs";
+
+namespace FBS.Response;
+
+union Body {
+  FBS.Worker.WorkerDumpResponse,
```

Comment:
This should be FBS.Worker.DumpResponse (and so on).

Reply:
Not possible in flatbuffers version I'm using, there is a name collision when different namespaces use the same name for a type, which is a bug in fbs. I'll check with the latest version.

Yes, I should be using the latest but for that I need to create the corresponding meson wrap, and the one being used is very close to the latest.

Reply:
Done here b5d4a7e2e9dc73a59ef0ffd3a3680081f019e3ed.

It was not about updating the flatbuffers demo but about to not use the '-gen all' option in flatc, which would export all included types in a given file, producing name collisions.